### PR TITLE
Day 5. Homework

### PR DESCRIPTION
**Pull request by eclimov**
+ Controller upgrades
+ Template upgrades
+ Entity upgrades
+ Repository

### CODE REVIEW

File: src/DataFixtures/JobFixtures.php [hunk 1]
```diff
@@ -25,15 +25,15 @@ public function load(ObjectManager $manager): void
             $job->setCompany($faker->company);
             $job->setDescription($faker->paragraph);
             $job->setEmail($faker->email);
-            $job->setExpiresAt($faker->dateTime);
+            $job->setExpiresAt($faker->dateTimeBetween($startDate = '-20 days', $endDate = '+20 days'));
```
Comment (gregurco) [inline src/DataFixtures/JobFixtures.php]: no need to write in this way and to introduce 2 more variables, just write:
```
$job->setExpiresAt($faker->dateTimeBetween('-20 days', '+20 days'));
```

File: src/Repository/JobRepository.php [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\Job;
+use \Doctrine\ORM\EntityRepository;
+
+class JobRepository extends EntityRepository
```
Comment (gregurco) [inline src/Repository/JobRepository.php]: you created repository in SF3 way. See how it should be in SF4:
https://symfony.com/doc/current/doctrine.html#querying-for-objects-the-repository

File: src/Repository/JobRepository.php [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+     * @return Job[]
+     */
+    public function findActiveJobs(): array
+    {
+        return $this->createQueryBuilder('j')
+            ->where('j.expiresAt > :datetime')
+            ->andWhere('j.activated = :activated')
+            ->setParameters([
+                'datetime' => new \DateTime(),
+                'activated' => true
```
Comment (gregurco) [inline src/Repository/JobRepository.php]: add `,` at the last element in array too

**Review by gregurco — CHANGES_REQUESTED** [review]

**Comment by eclimov** [comment]
Fixed

**Pull request merged**